src/main/java/dev/snowdrop/release/App.java

```diff
@@ -183,10 +183,11 @@ public void listCVEs(
         reportStatus(cves);
         if (release) {
             if (Optional.ofNullable(token).isPresent()) {
-                String mdReport = cveReportSvc.buildMdReport(cves, "Version - " + Optional.ofNullable(version).orElse(" ALL OPEN"));
-                git.createGithubIssue(mdReport, "CVE " + (version != null ? "for " + version : "list"), "cve", token, "snowdrop/reports");
+                git.closeOldCveIssues("cve",token, CVEReportingService.CVE_REPORT_REPO_NAME, version);
```

Comment:
Note that the method names should also be changed:  `Cve` -> `CVE` but that shouldn't be done as part of this PR.